docs/spec/index.html

```diff
@@ -1,740 +1,952 @@
+        issue</a>, if you wish to collaborate
+      on this specification.
+    </p>
+
+    <p>
+      You may also reach out via the mailing list:
+      <a href="mailto:public-credentials@w3.org">public-credentials@w3.org</a>
+      (<a
+        href="mailto:public-credentials-request@w3.org?subject=subscribe">subscribe</a>,
+      <a
+        href="https://lists.w3.org/Archives/Public/public-credentials/">archives</a>)
+    </p>
+  </section>
+
+  <section class="informative">
+    <h2>Introduction</h2>
+
+    <p>
+      This specification describes how verifiable data can be shared securely
+      over an authenticated channel.
+      Specifically, a set of OpenAPI HTTP endpoints is defined, allowing
+      authenticated data senders and receivers to
+      exchange Verifiable Presentations.
+    </p>
+
+    <p>
+      In this interaction-model, the presenting party initiates an interaction,
+      notifying the receiver. The
+      presentation receiver responds to the presentation intention with
+      cryptographic material for completing the
+      presentation. This is used by the presenter for proving the Verifiable
+      Presentation, which finally passed to the
+      receiver.
+    </p>
+
+    <p>
+      The <a href="#api-spec">Open API Specification</a> section covers each
+      operation and provides example inputs and
+      expected responses.
+    </p>
+
+    <section class="informative">
+      <h3>CCG Standards Stack</h3>
+      <img src="resources/standards-stack.png" alt="standards-stack"
+        width="100%" />
+      <p>The above diagram illustrates how traceability-interop fits into the
+        larger family of W3C CCG standards.</p>
       <ul>
         <li>
-          MAJOR - significant changes rolled forward from the previous major
-          version. Major versions MAY include breaking or non-backwards
-          compatible changes
+          At the <em>Business</em> layer, supply chain actors are exchanging
+          data in support of exchanged goods and
+          services.
+        </li>
+        <li>The <em>Trust</em> layer is supported by actors signing and
+          verifying data as Verifiable Credentials.</li>
+        <li>
+          The <em>Process</em> layer defines standardized APIs for exchanging
+          Verifiable Credentials (VC-API) and
+          Verifiable Presentations (trace-interop).
         </li>
         <li>
-          MINOR - backwards compatible changes that may introduce new
-          functionality or extensions of objects that are backwards compatible
+          <em>Linked Data</em> establishes a concise semantic context of the
+          data exchanged, enabled with JSON-LD by
+          the trace-vocab specification.
         </li>
         <li>
-          PATCH - minor changes that are non breaking and resolve discovered
-          issues or bugs
+          <em>Semantics</em> are defined in a multitude of established
```

Comment:
why the need to this? we have the vocab reference above, seems unnecessary to add more.

Reply:
agreed - this text was already present - opening issue #504 to cover these kinds of redundancies